RELEASING.md

```diff
@@ -12,76 +33,130 @@ export NEW_RELEASE=0.4.0.dev4
 poetry version $NEW_RELEASE
 ```
 
-## Update dependencies and run tests
+## Update dependencies
 
 ```bash
-
+poetry install --all-extras --sync
 poetry update
-pre-commit run --all-files && pytest kasa
 ```
 
-## Write a short and understandable summary for the release.
+## Run pre-commit and tests
 
-This step is only needed, if you want to include a summary text in the autogenerated changelog.
+```bash
+pre-commit run --all-files
+pytest kasa
+```
+
+## Create release summary (skip for dev releases)
+
+Write a short and understandable summary for the release.  Can include images.
 
-* Create a new issue and label it with release-summary
-* Create $NEW_RELEASE milestone in github, and assign the issue to that
-* Close the issue
+### Create $NEW_RELEASE milestone in github
 
-3. Generate changelog
+If not already created
+
+### Create new issue linked to the milestone
 
 ```bash
-# gem install github_changelog_generator --pre
-# https://github.com/github-changelog-generator/github-changelog-generator#github-token
-export CHANGELOG_GITHUB_TOKEN=token
-github_changelog_generator --base HISTORY.md --user python-kasa --project python-kasa --since-tag $PREVIOUS_RELEASE --future-release $NEW_RELEASE -o CHANGELOG.md
+gh issue create --label "release-summary" --milestone $NEW_RELEASE --title "$NEW_RELEASE Release Summary" --body "Some summary text"
+```
+
+You can exclude the --body option to get an interactive editor or leave blank and go into the issue on github and edit there.
+
+### Close the issue
+
+Either via github or:
+
+```bash
+gh issue close ISSUE_NUMBER
+```
 
+## Generate changelog
 
-#-exclude-tags-regex 'dev\d$'
+### For pre-release
+
+EXCLUDE_TAGS will exclude all dev tags except for the current release dev tags.
+
+Regex should be something like this `^((?!0\.7\.0)(.*dev\d))+`. The first match group negative matches on the current release and the second matches on releases ending with dev.
+
+```bash
+EXCLUDE_TAGS=${NEW_RELEASE%.dev*}; EXCLUDE_TAGS=${EXCLUDE_TAGS//"."/"\."}; EXCLUDE_TAGS="^((?!"$EXCLUDE_TAGS")(.*dev\d))+"
```

Comment:
```suggestion
export EXCLUDE_TAGS=${NEW_RELEASE%.dev*}; export EXCLUDE_TAGS=${EXCLUDE_TAGS//"."/"\."}; export EXCLUDE_TAGS="^((?!"$EXCLUDE_TAGS")(.*dev\d))+"
```
These need to be exported.

Reply:
I don't think they do as they're not used by any sub processes

Reply:
Ohhh, hmm, didn't think about it that way. So it's working as expected?

Reply:
Yes, it works